src/NUnitFramework/tests/Assertions/TypeAssertTest.cs

```diff
@@ -73,6 +73,7 @@ public void IsNotInstanceOf()
         }
 
         [Test]
+        [SetUICulture("en-US")]
         public void IsNotInstanceOfFails()
```

Comment:
Failure reason: Test asserts a constant English exception message that doesn't match the localized exception message.